### DISCUSSION TITLE
Blue line next to various issues in milestone

### DISCUSSION THREAD

**Discussion by sbrannen** [community/community#21885] [New to GitHub]
<p>What does a vertical blue line to the left of certain issues within a Milestone page signify?</p>
<p>I’ve searched and searched and cannot figure out why some issues are _marked_ blue.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/sbrannen">@sbrannen</a>,</p>
<p>I don’t have a repository with Milestones set up currently which is showing this. Would you mind sharing a screenshot of what you’re seeing so that we can better answer your question?</p>
<p>Thanks!</p>

**Comment by sbrannen** [comment]
<p>Milestones in the JUnit 5 repository almost always display some issues with a blue bar to the left.</p>
<p>I see two there right now: <a href="https://github.com/junit-team/junit5/milestone/22" rel="nofollow noopener">https://github.com/junit-team/junit5/milestone/22</a></p>

**Comment by sbrannen** [comment, marked as answer]
<p>A colleague solved the mystery for me: the blue lines indicate that you have an unread notification associated with that issue. So, other people don’t necessarily see the blue lines just because you do.</p>
<p>If you want to see these blue lines for yourself, just check out the animated GIF here: <a href="https://help.github.com/articles/about-milestones/#prioritizing-issues-and-pull-requests-in-milestones" rel="nofollow noopener">https://help.github.com/articles/about-milestones/#prioritizing-issues-and-pull-requests-in-milestones</a></p>
